mantle/kola/register/register.go

```diff
@@ -79,6 +79,10 @@ type Test struct {
 	// FailFast skips any sub-test that occurs after a sub-test has
 	// failed.
 	FailFast bool
+
+	// If true, this test will be run along with other NonExclusive tests in one VM
+	// Otherwise, it is run in its own VM
+	NonExclusive bool
```

Comment:
```suggestion
	Exclusive bool
```

For some reason I'd still use `Exclusive` right here instead.